hi.js

```diff
@@ -220,11 +220,11 @@ settings.interface.nametagStyle.all सब कुछ
 settings.interface.healthNum नैम-टैग HP संख्या
 settings.interface.healthColT नैम-टैग अपनी टीम के HP का रंग
 settings.interface.healthColE नैम-टैग दुश्मन के HP का रंग
-settings.interface.xpBarType XP Bar Type
+settings.interface.xpBarType XP बार का टाइप
 settings.interface.xpBarCol XP बार का रंग
 settings.interface.xpBarCol2 XP Bar Color 2
 settings.interface.xpBarOpac XP बार की पारदर्शकता
-settings.interface.adsObjOpac ADS Objective Opacity
+settings.interface.adsObjOpac ADS ऑब्जेक्टिव अपारदर्शिता
```

Comment:
अपारदर्शिता too complicated, leave the term as it is

Reply:
so opacity in hindi or just opacity,

Reply:
i took that word from other opacity translations